### PR TITLE
Update master to match v0.14.0, prep for v0.14.1

### PR DESCRIPTION
**Pull request by fluffypony**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,9 +123,10 @@ Dates are provided in the format YYYY-MM-DD.
...
 | 1546000                        | 2018-04-06 | v7                | v0.12.0.0              | v0.12.3.0                  | Cryptonight variant 1, ringsize >= 7, sorted inputs
-| 1685555                        | 2018-10-18 | v8                | v0.13.0.0              | v0.13.0.4                  | max transaction size at half the penalty free block size, bulletproofs enabled, cryptonight variant 2, fixed ringsize [11](https://youtu.be/KOO5S4vxi0o)
-| 1686275                        | 2018-10-19 | v9                | v0.13.0.0              | v0.13.0.4                  | bulletproofs required
-| XXXXXXX                        | 2019-04-XX | XX                | XXXXXXXXX              | XXXXXXXXX                  | X
+| 1685555                        | 2018-10-18 | v8                | v0.13.0.0              | v0.13.0.0                  | max transaction size at half the penalty free block size, bulletproofs enabled, cryptonight variant 2, fixed ringsize [11](https://youtu.be/KOO5S4vxi0o)
+| 1686275                        | 2018-10-19 | v9                | v0.13.0.0              | v0.13.0.0                  | bulletproofs required
```
Comment (moneromooo-monero) [inline README.md]: Why did you remove 0.13.0.4, is it bad somehow ?

File: README.md [hunk 2]
```diff
@@ -123,9 +123,10 @@ Dates are provided in the format YYYY-MM-DD.
...
+| 1788000                        | 2019-03-09 | v10               | v0.14.0.0              | v0.14.0.0                  | Cryptonight-R PoW, new block weight algorithm, slightly more efficient RingCT format
+| 1788720                        | 2019-03-10 | v11               | v0.14.0.0              | v0.14.0.0                  | forbid old RingCT transaction format
```
Comment (moneromooo-monero) [inline README.md]: 0.14.0.2 on the last column.

File: src/cryptonote_core/blockchain.cpp [hunk 3]
```diff
@@ -4697,8 +4697,8 @@ void Blockchain::cancel()
 }
 
 #if defined(PER_BLOCK_CHECKPOINT)
-static const char expected_block_hashes_hash[] = "954cb2bbfa2fe6f74b2cdd22a1a4c767aea249ad47ad4f7c9445f0f03260f511";
-void Blockchain::load_compiled_in_block_hashes(const GetCheckpointsCallback& get_checkpoints)
+static const char expected_block_hashes_hash[] = "570ce2357b08fadac6058e34f95c5e08323f9325de260d07b091a281a948a7b0";
+void Blockchain::load_compiled_in_block_hashes()
```
Comment (moneromooo-monero) [inline src/cryptonote_core/blockchain.cpp]: That prototype change looks like a merge error ?

File: README.md (hunk 1 above)
Comment (fluffypony) [reply README.md]: WHOOPS! I'll fix now.

File: README.md (hunk 2 above)
Comment (fluffypony) [reply README.md]: Minimum Monero version is 0.14.0.0, no?

File: src/cryptonote_core/blockchain.cpp (hunk 3 above)
Comment (fluffypony) [reply src/cryptonote_core/blockchain.cpp]: Thanks, fixing now

File: README.md (hunk 2 above)
Comment (moneromooo-monero) [reply README.md]: Yes, second to last column is good, last column is "recommended", and you want 0.14.0.2 there, due to the problems we've just fixed.

Comment (fluffypony) [reply README.md]: Nevermind, I can't read today. This is in prep for 0.14.1 so last column should match that, right?

Comment (moneromooo-monero) [reply README.md]: Well, given there won't be a 0.14.1.0 tag for a while... the recommended one is still 0.14.0.2 I think.
